### PR TITLE
Add web links to RSS

### PR DESCRIPTION
**Pull request by petrjasek**
SDFID-592

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -35,6 +35,6 @@ install:
     - cd client && npm install && cd ..
 
 script:
-    - cd server && flake8 && cd ..
+    - cd server && flake8 && pytest && behave && cd ..
```
Comment (ride90) [inline .travis.yml]: Are we going to use pytest to run tests for fidelity?

File: server/fidelity/content_api_rss.py [hunk 2]
```diff
@@ -14,11 +16,23 @@
...
 
 def get_permalink(item):
-    return "{}/{}".format(
-        flask.url_for("rss.index", _external=True).rstrip("/"), item["_id"]
+    code = item["_id"][-6:]
+    try:
+        title = item["extra"][PERMALINK] or ""
+        slug = slugify(title)
+    except (KeyError, AttributeError):
+        title = ""
```
Comment (ride90) [inline server/fidelity/content_api_rss.py]: Why do we need an empty title variable here?

Comment (petrjasek) [reply server/fidelity/content_api_rss.py]: they don't set it for translated items

File: .travis.yml (hunk 1 above)
Comment (petrjasek) [reply .travis.yml]: I've just enabled it, seems simpler

**Pull request merged**